src/document.c

```diff
@@ -1656,6 +1655,12 @@ gboolean document_reload_prompt(GeanyDocument *doc, const gchar *forced_enc)
 	g_free(base_name);
 	return result;
 }
+/* also used for reloading when forced_enc is NULL */
+gboolean document_rename_prompt(GeanyDocument *doc) {
+	document_show_tab(doc);
+	dialogs_show_save_as();
+	return FALSE;
```

Comment:
why return something if it's always `FALSE`?  if anything, it should be `return dialogs_show_save_as();`